layouts/shortcodes/artifact.html

```diff
@@ -0,0 +1,27 @@
+{{- $org := trim (.Get "org") " " -}}
+{{- if not $org -}}
+{{- $org = "knative" -}}
+{{- end -}}
+{{- $repo := trim (.Get "repo") " " -}}
+{{- $file := trim (.Get "file") " " -}}
+{{- $pageSection := .Page.Section -}}
+{{- $preRelease := .Site.Params.masterfolder -}}
+{{- if ne $pageSection $preRelease -}}
```

Comment:
Is this a copy of the version shortcode, maybe we can just call that from here? https://gohugo.io/content-management/shortcodes/#nested-shortcodes

Reply:
It's a modified form of it, since I need different behavior in the two different paths (vs. defaulting).  I'm not sure that my kung fu is strong enough to refactor this yet.